internal/providers/terraform/azure/cdn_endpoint.go

```diff
@@ -0,0 +1,301 @@
+package azure
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/infracost/infracost/internal/usage"
+	"github.com/shopspring/decimal"
+	log "github.com/sirupsen/logrus"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMCDNEndpointRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_cdn_endpoint",
+		RFunc: NewAzureCDNEndpoint,
+		ReferenceAttributes: []string{
+			"profile_name",
+		},
+	}
+}
+
+func NewAzureCDNEndpoint(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	var costComponents []*schema.CostComponent
+
+	profile := d.References("profile_name")[0]
+	sku := profile.Get("sku").String()
+	location := locationToZone(d.Get("location").String())
+	if location == "" {
+		log.Warnf("Unrecognized CDN location for resource %s: %s", d.Address, sku)
+		return nil
+	}
+
+	if len(strings.Split(sku, "_")) != 2 || sku == "Standard_ChinaCdn" {
+		log.Warnf("Unrecognized/unsupported CDN sku format for resource %s: %s", d.Address, sku)
+		return nil
+	}
+
+	costComponents = append(costComponents, cdnOutboundDataCostComponents(location, sku, u)...)
+
+	if sku == "Standard_Microsoft" {
```

Comment:
@alikhajeh1 is this a place where we should be doing a case-insensitive comparison?  i.e.:
```suggestion
	if strings.ToLower(sku) == "standard_microsoft" {
```

Reply:
Yep, anytime we see `== "` in our code, we should be thinking about case-sensitive comparisons...